cmd/microcloud/cluster_members.go

```diff
@@ -1,16 +1,47 @@
 package main
 
 import (
+	"bufio"
 	"context"
+	"fmt"
+	"io"
+	"os"
 	"sort"
+	"strconv"
+	"strings"
 
 	"github.com/canonical/lxd/shared"
 	cli "github.com/canonical/lxd/shared/cmd"
+	"github.com/canonical/lxd/shared/termios"
 	"github.com/canonical/microcluster/client"
+	"github.com/canonical/microcluster/cluster"
 	"github.com/canonical/microcluster/microcluster"
 	"github.com/spf13/cobra"
+	"golang.org/x/sys/unix"
+	"gopkg.in/yaml.v2"
```

Comment:
Could you please fix these imports to be structured in 3 blocks: 
```
built-in packages
external packages
internal packages
```

Reply:
Do we consider microcluster & LXD to be internal? It's not structured that way [in other files](https://github.com/canonical/microcloud/blob/359cd469a36f52a950f24ec9945f18f22e74d039/cmd/microcloud/main_init.go#L13-27)

Reply:
Yeah this is kind of a "fix it as we see it" sort of thing. Any import that is not from the immediate project (so `microcloud`) is considered external. LXD captures this with [golangci-lint](https://github.com/canonical/lxd/blob/main/test/lint/golangci.sh) on only files that were touched by the commits.

You may have seen the following error from static analysis in LXD:
```
File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(github.com/canonical/lxd) (gci)
```


Specifically `golangci-lint` checks the imports using https://github.com/daixiang0/gci so if you were to run it:
```bash
PROJECT="github.com/canonical/microcloud"
FILES="$(git diff upstream/main --name-only)

gci diff --skip-generated -s standard -s default -s "prefix(${PROJECT})" ${FILES}
```

Reply:
It's possible I'm just really blind but it looks to me like I'm already following that convention here... there aren't any microcloud imports, just microcluster ones.

Reply:
Nope, I'm the blind one `microcl___` is all I see.